docs/fidesops/docs/guides/saas_connectors.md

```diff
@@ -0,0 +1,109 @@
+# How-To: Connect to SaaS Applications
+
+In this section we'll cover
+
+- What is a SaaS connection?
+- What is a SaaS config schema?
+- How do you configure a SaaS connection?
+
+## What is a SaaS connection?
+
+A SaaS (Software as a Service) connection is a connection type within Fidesops that allows a user to connect to a SaaS application (e.g., Mailchimp, Stripe, Slack, etc.) and execute data access and erasure requests against that application. These connections use functionality introduced in earlier sections (ConnectionConfigs, Datasets, Policies) but also introduce a new SaaS config specification to define how to connect to specific SaaS applications.
```

Comment:
It's an aside but I do believe we'll need to move away from SaaS specific naming at a later point. In the future these connectors won't always be SaaS specific. They'll more likely be generic third party API integrations of some sort.

Reply:
Agreed, SaaS connectors is the name @NevilleS and I settled on but it's not set in stone. We can have another brainstorming session around this.